src/main/java/appeng/core/localization/ButtonToolTips.java

```diff
@@ -64,11 +64,15 @@
 
 	TransferToStorageCell,
 	ToggleSortDirection,
-	SearchMode_Auto,
 
+	SearchMode_Auto,
 	SearchMode_Standard,
 	SearchMode_JEIAuto,
 	SearchMode_JEIStandard,
+	SearchMode_AutoKeep,
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)